dataactvalidator/config/sqlrules/c14_award_financial_2.sql

```diff
@@ -0,0 +1,5 @@
+SELECT row_number, fain, uri, piid
```

Comment:
Even though the original C14 was pretty simple, I like the idea of having this logic as a separate query to enable a more specific error message.